.ci/azure/windows_conditional_compilation.yml

```diff
@@ -95,16 +109,27 @@ jobs:
   - script: dir $(REPO_DIR)\temp\ /s
     displayName: 'List temp SDKs'
 
-  - script: call "$(MSVS_VARS_PATH)" && $(WORK_DIR)\ninja-win\ninja
+  - script: |
+      call "$(MSVS_VARS_PATH)" && cmake --build . --config $(BUILD_TYPE) && cmake --build . --config $(BUILD_TYPE) --target sea_itt_lib
     workingDirectory: $(BUILD_DIR)
-    displayName: 'Build Win CC'
+    displayName: 'Build Win CC COLLECT'
 
   - script: dir $(REPO_DIR)\bin\ /s
     displayName: 'List bin files'
 
-  - script: cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -P cmake_install.cmake
-    workingDirectory: $(BUILD_DIR)
-    displayName: 'Install'
+  - script: |
+      call "$(MSVS_VARS_PATH)" && python thirdparty\itt_collector\runtool\sea_runtool.py --bindir $(REPO_DIR)\bin\intel64\$(BUILD_TYPE) -o $(BUILD_DIR)\itt_stat ! $(REPO_DIR)\bin\intel64\$(BUILD_TYPE)\benchmark_app.exe -niter 1 -nireq 1 -m $(MODELS_PATH)\models\test_model\test_model_fp32.xml
+    workingDirectory: $(REPO_DIR)
+    displayName: 'Code usage analysis'
+
+  - script: |
+      call "$(MSVS_VARS_PATH)" && cmake -G"Visual Studio 16 2019" -DVERBOSE_BUILD=ON -DENABLE_FASTER_BUILD=ON -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DENABLE_PROFILING_ITT=OFF -DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=$(BUILD_DIR)\*.csv -DCMAKE_C_COMPILER:PATH="$(MSVC_COMPILER_PATH)" -DCMAKE_CXX_COMPILER:PATH="$(MSVC_COMPILER_PATH)" $(REPO_DIR)
+    workingDirectory: $(BUILD_DIR_2)
+    displayName: 'CMake2'
 
-  - script: dir $(INSTALL_DIR) /s
-    displayName: 'List install files'
+  - script: cmake --build . --config $(BUILD_TYPE)
+    workingDirectory: $(BUILD_DIR_2)
+    displayName: 'Build WinCC ON'
```

Comment:
should we also add a step that BA can run this topology?

Reply:
BTW, Win CC now takes 100 mins. Can we turn off unused plugins and other functionality like frontends where CC does not make sence

Reply:
Disabled the last check due to the issue on CPU. 
This step should be enabled after the fix

Reply:
> Disabled the last check due to the issue on CPU.

Do we have a ticket for this issue?